doc/gauss_newton.rst

```diff
@@ -0,0 +1,45 @@
+..
+    : An experimental local optimization package
+    : Copyright (C) 2018 Ayers Lab <ayers@mcmaster.ca>.
+    :
+    : This file is part of Flik.
+    :
+    : Flik is free software; you can redistribute it and/or
+    : modify it under the terms of the GNU General Public License
+    : as published by the Free Software Foundation; either version 3
+    : of the License, or (at your option) any later version.
+    :
+    : Flik is distributed in the hope that it will be useful,
+    : but WITHOUT ANY WARRANTY; without even the implied warranty of
+    : MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    : GNU General Public License for more details.
+    :
+    : You should have received a copy of the GNU General Public License
+    : along with this program; if not, see <http://www.gnu.org/licenses/>
+
+Gauss–Newton Algorithm
+======================
+
+The advantage of Gauss-Newton Method is that the second derivatives are not necessarily needed to be
+computed.The limitation is also obvious since it can only handle with the minimization the sum of
```

Comment:
- space before The. 
- "since it can only minimize the sum of squared function values".

Reply:
1. Why is space needed? When I compile the website, it looks like something unusual.
2. Fixed.